src/devtran.cpp

```diff
@@ -716,7 +721,11 @@ Rcpp::List DEVTRAN(const Rcpp::List parin,
         used_mtimehx = mtimehx.size() > 0;
         prob.clear_mtime();
       } // Close handling of modeled events
-
+      
+      if(call_event && !this_rec->is_lagged()) {
+        prob.table_call();
+      }
+      
       if(this_rec->output()) {
```

Comment:
Second bit of "workaround" code to maintain event creation in both `$EVENT` and `$TABLE`.